### DISCUSSION TITLE
Sentence split into multiple lines

### DISCUSSION THREAD

**Discussion by shmightworks** [WinMerge/winmerge#1022] [Q&A]
Seems simple but can't find the option to ignore this.

Say you have:
This is a test.

Then you have:
This is
a test.

Anyway to ignore this as different?
I thought the whitespace might take care of this, but doesn't.

I get lots of these differences because the text is copied from a pdf, and with wrapping, it adds new line all over the place in different places.

**Comment by sdottaka** [comment]
Unfortunately, WinMerge does not handle such cases well.

In WinMerge version 2.16.14, the plugin's functionality has been slightly improved to allow text to be processed so that unwanted differences do not appear.

The following example uses the Tokenize unpacker plugin and ExecFilterCommand unpacker plugin. 
(The result of processing this text may not be easy to read) 

~~~
Tokenize [^\w]+
~~~
~~~
ExecFilterCommand powershell -c "$(-join $input) -replace '\. ', "".`r`n"""
~~~

![#1022_2](https://user-images.githubusercontent.com/98126/138534406-89bbfc53-a259-4ce5-858b-823b23ea2003.gif)
